src/plugins/inputs/whois.js

```diff
@@ -1,15 +1,39 @@
 "use strict";
+const Msg = require("../../models/msg");
 
-exports.commands = ["whois"];
+exports.commands = ["whois", "whowas"];
 
 exports.input = function ({irc}, chan, cmd, args) {
-	if (args.length === 1) {
-		// This queries server of the other user and not of the current user, which
-		// does not know idle time.
-		// See http://superuser.com/a/272069/208074.
-		irc.raw("WHOIS", args[0], args[0]);
-	} else {
-		// Re-assembling the command parsed in client.js
-		irc.raw(`${cmd} ${args.join(" ")}`);
+	const client = this;
+	const target = args[0];
+	const targetNick = args[1] ? args[1] : target;
```

Comment:
This I don't understand, args[0] should be the nick.
Why are you using args[1] ?

That made sense originally when we were querying the idle time, but we aren't doing that anymore.

Reply:
Which probably brings me to the second point, should we add that back? Chances are yes right? Because we do want the idle time?

Reply:
I kept the idle time code, but forgot to whois the nick and targetNick. Fixed.